src/Ng/CommandHelpers.cs

```diff
@@ -34,6 +92,17 @@ static class CommandHelpers
             for (int i = 1; i < args.Length; i += 2)
             {
                 result.Add(args[i], args[i + 1]);
+                inputArgs.Add(args[i]);
```

Comment:
>inputArgs [](start = 16, length = 9)

we don't need a second data structure. You can just go over the original dictionary, and return a new dictionary.